tests/e2e/utils/playwright-setup.js

```diff
@@ -122,7 +136,11 @@ export const createApiTokens = ( page ) =>
 	} );
 
 /**
- * Helper function to update the plugin.
+ * Helper function to download the Stripe plugin from the repository and install it on the site.
+ * This is useful when we want to test a specific version of the plugin.
+ * If the plugin is already installed, it will be updated to the specified version.
+ * @param {Page} page Playwright page object.
```

Comment:
Feel free to ignore: In some places we're using `{Page} page`, and in other places `{Object} page`.